tools/model_tools/src/openvino/model_zoo/omz_quantizer.py

```diff
@@ -137,6 +48,9 @@ def main():
     parser.add_argument('--target_device', help='target device for the quantized model')
     args = parser.parse_args()
 
+    if not args.dataset_dir:
+            sys.exit('--dataset_dir must be specified.')
+
```

Comment:
If you move this to `Quantizer` it's not needed here anymore, it will brake `--print_all` option.